### ISSUE TITLE
Version 2 has broken our build

### ISSUE THREAD

**Issue by mzedeler** [woodpecker-ci/plugin-git#51]
After the update to version 2, we get this error when Woodpecker starts:

```
+ git init -b master
Initialized empty Git repository in /drone/src/github.com/<snip>/.git/
+ git remote add origin https://github.com/<snip>
+ git fetch --no-tags --depth=1 --filter=tree:0 origin +refs/heads/master:
fatal: could not read Username for 'https://github.com': terminal prompts disabled
exit status 128
```

**Comment by lafriks** [comment]
Same is also for me, works fine with v1.6.1

**Comment by lafriks** [comment]
Should be fixed by #52
